tests/test_zenhub.py

```diff
@@ -30,6 +30,23 @@ def json(self):
             200,
             'Ok'
         )
+
+    # Issue events
+    elif args == ('https://api.zenhub.io/p1/repositories/123456789/issues/42/events',) and \
+            kwargs == {'headers': {'X-Authentication-Token': '99999999', 'Content-Type': 'application/json'}}:
+        return MockResponse(
+            [{'created_at': '2019-05-08T22:13:43.512Z',
+              'from_estimate': {'value': 8},
+              'to_estimate': {'value': 4},
+              'type': 'estimateIssue'},
+             {'created_at': '2019-04-20T14:12:40.900Z',
+              'from_estimate': {'value': 4},
+              'to_estimate': {'value': 8},
+              'type': 'estimateIssue'}],
+            200,
+            'Ok'
```

Comment:
I think it's `OK`, but check it before you change it.

Reply:
Good catch!